sample-apps/react-native/dogfood/src/screens/AudioRoom/Room.tsx

```diff
@@ -18,6 +19,12 @@ export default function Room({ onClose }: { onClose: () => void }) {
   const callingState = useCallCallingState();
   const call = useCall();
 
+  // Automatically route audio to speaker devices as relevant for watching videos.
+  useEffect(() => {
+    InCallManager.start({ media: 'audio' });
+    return () => InCallManager.stop();
+  }, []);
```

Comment:
comment says route to speaker.. but media is audio..

Reply:
Let me fix this